### PR TITLE
Merge develop

### LINKED ISSUE #8: Online/offline events support
https://developer.mozilla.org/en-US/docs/Online_and_offline_events
http://html5demos.com/offline

But:
http://stackoverflow.com/questions/3181080/how-to-detect-online-offline-event-cross-browser

### LINKED ISSUE #9: port to cerebral@0.34
see following for refs:
cerebral/cerebral-module-router#86
cerebral/cerebral-module-recorder#7
cerebral/cerebral-module-signal-store#5

### PR DESCRIPTION
**Pull request by maxfrigge**
Resolves #8 
Resolves #9

### CODE REVIEW

File: demo/package.json [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
     "baobab": "^2.3.3",
-    "cerebral": "^0.33.19",
+    "cerebral": "^0.34.0-rc.2",
     "cerebral-model-baobab": "^0.4.7",
-    "cerebral-module-devtools": "^0.5.3",
+    "cerebral-module-devtools": "^0.7.0-rc.1",
     "cerebral-view-react": "^0.11.2",
-    "react": "^0.14.7",
-    "react-dom": "^0.14.7"
+    "react": "^15.0.1",
```
Comment (Guria) [inline demo/package.json]: this bad one

**Comment by Guria** [comment]
cc @christianalfoni

**Comment by christianalfoni** [comment]
This looks good to me! :-)

**Comment by Guria** [comment]
@christianalfoni thx, waiting release before merge

**Pull request merged**